auth-web/src/components/auth/manage-business/AddBusinessDialog.vue

```diff
@@ -1,104 +1,140 @@
 <template>
-  <div id="add-business-form">
+  <div id="add-business-dialog">
     <HelpDialog
       :helpDialogBlurb="helpDialogBlurb"
       ref="helpDialog"
     />
 
-    <v-form ref="addBusinessForm" lazy-validation>
-      <!-- Business Identifier -->
-      <v-text-field
-        filled req persistent-hint
-        label="Incorporation Number or Registration Number"
-        hint="Example: BC1234567, CP1234567 or FM1234567"
-        :rules="businessIdentifierRules"
-        v-model="businessIdentifier"
-        @blur="formatBusinessIdentifier()"
-        class="business-identifier mb-n2"
-        aria-label="Incorporation Number and Password or Passcode"
-      />
-
-      <!-- Passcode -->
-      <v-expand-transition>
-        <v-text-field
-          v-if="isBusinessIdentifierValid"
-          filled
-          :label="passcodeLabel"
-          :hint="passcodeHint"
-          persistent-hint
-          :rules="passcodeRules"
-          :maxlength="passcodeMaxLength"
-          v-model="passcode"
-          autocomplete="off"
-          class="passcode mt-6 mb-n2"
-          :aria-label="passcodeLabel"
-        />
-      </v-expand-transition>
-
-      <!-- Certify (firms only) -->
-      <v-expand-transition>
-        <Certify
-          v-if="isBusinessIdentifierValid && isFirm"
-          :clause="certifyClause"
-          entity="registered entity"
-          @update:isCertified="isCertified = $event"
-          class="certify mt-6"
-        />
-      </v-expand-transition>
-
-      <!-- Folio Number -->
-      <v-expand-transition>
-        <section v-if="isBusinessIdentifierValid" class="mt-6">
-          <header class="font-weight-bold">Folio / Reference Number</header>
-          <p class="mt-4 mb-0">
-            If you file forms for a number of companies, you may want to enter a
-            folio or reference number to help you keep track of your transactions.
+    <v-dialog
+      attach="#entity-management"
+      v-model="dialog"
+      persistent
+      scrollable
```

Comment:
Only the v-card text scrolls. The title and actions stay in place. Example:

![image](https://user-images.githubusercontent.com/10002889/182977754-96410b50-14ba-4dbf-b075-221bf5600408.png)

Reply:
These are great improvements.